pvr.dvbviewer/addon.xml.in

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.dvbviewer"
-  version="2.0.1"
+  version="2.0.2"
```

Comment:
There was one which bumped from 1.11.10 to 2.0.0. But it's probably better to bump to 3.1.0 here. thank you